### ISSUE TITLE
Backspace Key in Editor Doesn't Work

### ISSUE THREAD

**Issue by volomike** [anuraghazra/BugVilla#18]
I created some text, then moved my cursor back into the editor near the top, and then clicked the Backspace key. Instead of removing characters, it appended some. I had to use the Forward Delete key to delete characters, instead.

**Comment by anuraghazra** [comment]
Yes I'm aware of this bug. The Editor has these kind of bugs which I could not reproduce nor figured out the cause. 

Your suggestions in #19 might solve the problem

**Comment by harshilparmar** [comment]
Can I try this?

**Comment by anuraghazra** [comment]
Yes of course @harshilparmar you are more than welcome :D

**Comment by harshilparmar** [comment]
bro I mailed you please check @anuraghazra
